src/packages/@ncigdc/modern_components/SsmsTable/SsmsTable.js

```diff
@@ -247,14 +251,9 @@ export default compose(
           data={frequentMutations.map(({ score = 0, ...x }) => ({
             ...x,
             mutation_uuid: (
-              <span>
-                <MutationLink uuid={x.ssm_id}>
-                  {x.ssm_id.substr(0, 8)}
-                </MutationLink>
-                <ForTsvExport>
-                  {x.ssm_id}
-                </ForTsvExport>
-              </span>
+              <ForTsvExport>
```

Comment:
the column is only for tsv export, the Th is `display: none`, could do `display: none` on span but went with `ForTsvExport` so it's clearly there only for export.